Documentation/getting-started-test-app.md

```diff
@@ -31,6 +95,20 @@ In other Android environments, the ad ID tracking authorization is controlled us
 [<img src="./assets/new_adid_setting_optin.png" alt="New ad ID settings page - opt-in state" width="210"/>](./assets/new_adid_setting_optin.png)
 [<img src="./assets/new_adid_setting_optout.png" alt="New ad ID settings page - opt-out state" width="210"/>](./assets/new_adid_setting_optout.png)
 
+## Validation with Assurance
+
+Configure a new Assurance session by setting the Base URL to `testapp://main` and launch Assurance in the demo app by running the following command in your terminal:
+
+```bash
+$ adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=ADD_YOUR_SESSION_ID_HERE" com.adobe.marketing.mobile.testapp
+```
+
+Note: replace `ADD_YOUR_SESSION_ID_HERE` with your Assurance session identifier.
+
+Once the connection is established and the events list starts getting populated, you can filter the events for this extension by typing `Edge Identity` in the `Search Events` search box.
+
+
+
 ## Android Ads SDKs
```

Comment:
these sections need to be under Testing tips for Android advertising identifier, so use h4 / h5 for these titles